### PR TITLE
[doc/roadmap]

### PR DESCRIPTION
**Pull request by vertexportus**
roadmap in a separate .md file

### CODE REVIEW

**Comment by vertexportus** [comment]
@rafapaulin this is a comment

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## 1.0
```
Comment (rafapaulin) [inline ROADMAP.md:3]: Insert a header here to keep the pattern (similar to CHANGELOG.md)

**Review by rafapaulin — CHANGES_REQUESTED** [review]
- Need to update the Table of contents to point to the external file (similar to the CHANGELOG.md)
- Need to copy the files (README.md, ROADMAP.md) from the root of the project (read by github) to the projects/smarti18n (used by the cli to build the lib)

**Review by rafapaulin — CHANGES_REQUESTED** [review]
- ROADMAP.md still missing # ROADMAP title
- need a copy of the roadmap inside projects/smarti18n

**Review by rafapaulin — APPROVED** [review]

**Pull request merged**